blocks/models/huggingface/base.py

```diff
@@ -75,9 +75,8 @@ def load(self) -> None:
 
     def fit(self, dataset: List[str], labels: Optional[pd.Series]) -> None:
         assert (
-            self.model is not None,
-            "Either a trained model or a pretrained model must be loaded.",
-        )
+            self.model is not None
```

Comment:
previously we asserted the existence of a tuple, which was always True. now we have the condition, then the text